clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -12366,7 +12366,7 @@ def err_preserve_enum_value_not_const: Error<
   "__builtin_preserve_enum_value argument %0 not a constant">;
 
```

Comment:
It would probably be nice if someone did a general cleanup pass on all the diagnostics TD files, because I just saw a bunch more up here that aren’t quoted, but that sounds like a lot of work...

~~Maybe we can tag that as a good first issue?~~ ;Þ